src/lib/Command/IbexaSetupCommand.php

```diff
@@ -33,8 +35,7 @@ class IbexaSetupCommand extends BaseCommand
 
     protected function configure(): void
     {
-        $this->setName('ibexa:setup')
-             ->setDescription('Runs post install configuration tool.')
+        $this
```

Comment:
This could be merged into one line with lines 39 and 40.

Reply:
Change has been applied manually